app/grandchallenge/reader_studies/models.py

```diff
@@ -926,6 +926,24 @@ class AnswerType(models.TextChoices):
     ANGLE = "ANGL", "Angle"
     MULTIPLE_ANGLES = "MANG", "Multiple angles"
 
+    @staticmethod
+    def annotation_types():
+        return [
+            AnswerType.BOUNDING_BOX_2D,
+            AnswerType.MULTIPLE_2D_BOUNDING_BOXES,
+            AnswerType.DISTANCE_MEASUREMENT,
+            AnswerType.MULTIPLE_DISTANCE_MEASUREMENTS,
+            AnswerType.POINT,
+            AnswerType.MULTIPLE_POINTS,
+            AnswerType.POLYGON,
+            AnswerType.MULTIPLE_POLYGONS,
+            AnswerType.MASK,
+            AnswerType.LINE,
+            AnswerType.MULTIPLE_LINES,
+            AnswerType.ANGLE,
+            AnswerType.MULTIPLE_ANGLES,
+        ]
+
```

Comment:
This change was necessary to use this in the answer filter for the PDF. Not sure why this needed to be a property of the Question model. Making it a staticmethod on the `AnswerType` enum seemed a little more logical to me. This is also how we do it for interface types, for example.